src/platform/site-wide/drupal-static-data/tests/connect/fetch.unit.spec.js

```diff
@@ -0,0 +1,39 @@
+import environment from 'platform/utilities/environment';
+
+const sinon = require('sinon');
+const { expect } = require('chai');
+const { fetchDrupalStaticDataFile } = require('../../connect/fetch');
+
+describe('fetchDrupalStaticDataFile', () => {
+  let mockFetch;
+
+  const jsonOK = body =>
+    new Response(JSON.stringify(body), {
+      status: 200,
+      headers: {
+        'Content-type': 'application/json',
+      },
+    });
+
+  beforeEach(() => {
+    mockFetch = sinon.stub(global, 'fetch').resolves(jsonOK(['test data']));
```

Comment:
**CONSIDER**
`sinon` offers mocks as well, so I wonder if we should call this variable `stubFetch` or `fetchStub` rather than `mockFetch`.

**MUSING**
I also wonder if it should _be_ a mock rather than a stub given our need to verify the arguments used to call it. This is probably not a critical distinction in a test suite like this, but my understanding is that mocks are more idiomatic than stubs when needing to verify interaction (e.g. arguments).

Reply:
I just discovered that there already is a `mockFetch` helper function within the repo at https://github.com/department-of-veterans-affairs/vets-website/blob/f91d4786d16a28825917a59ff8805e01ebb96014/src/platform/testing/unit/helpers.js#L111. It doesn't really offer anything more than what I created in the way of testing args though. Your idea to use a sinon mock instead of a stub actually shows more promise, and I'm refactoring based on that.